rpcperms/interceptor.go

```diff
@@ -413,6 +413,12 @@ func (r *InterceptorChain) rpcStateUnaryServerInterceptor() grpc.UnaryServerInte
 	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo,
 		handler grpc.UnaryHandler) (interface{}, error) {
 
+		if info.Server == nil {
+			return nil, fmt.Errorf("server is nil, can't check " +
```

Comment:
Can probably be a bit less specific in the error message here, perhaps just "cannot handle call, server not ready"?